agent.toml file dne breaks Storage.set after agent create completes successfully

```
➜  cli git:(main) ✗ go run *.go agent create      
-Calling Agent Create, with Address: 0x8b35624Ed57789D18445142a51A4a51eFb375F26, Address: 0x69D2CE31DDABF4A7098a2547147c13cF10F5Ea7b, Address: 0x115fFf27B67875032D6E6D2F28a3aAbC31A69f54
Agent create transaction submitted: 0xf1fbd9b0fb0a68872cf5d13ca5d00637bf4553fd192f9feba91e0edc58e74fcd
Waiting for confirmation...
Transaction receipt received.
Status: 1
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/glif-confidential/cli/util.(*Storage).Set(...)
        /Users/jonathanschwartz/Documents/glif/cli/util/storage.go:96
github.com/glif-confidential/cli/cmd.glob..func4(0x1036cadc0, {0x102bf126c?, 0x0?, 0x0?})
        /Users/jonathanschwartz/Documents/glif/cli/cmd/agent_create.go:93 +0x708
github.com/spf13/cobra.(*Command).execute(0x1036cadc0, {0x103711798, 0x0, 0x0})
        /Users/jonathanschwartz/.go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x5b0
github.com/spf13/cobra.(*Command).ExecuteC(0x1036ca800)
        /Users/jonathanschwartz/.go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x35c
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/jonathanschwartz/.go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/glif-confidential/cli/cmd.Execute()
        /Users/jonathanschwartz/Documents/glif/cli/cmd/root.go:43 +0x28
main.main()
        /Users/jonathanschwartz/Documents/glif/cli/main.go:21 +0x1c
```

--- a/util/storage.go
+++ b/util/storage.go
@@ -36,6 +36,13 @@ func NewStorage(filename string, defaultMap map[string]string) (*Storage, error)
 		if err != nil {
 			return nil, err
 		}
+		if len(s.data) == 0 {
+			s.data = defaultMap
+			err = s.save()
+			if err != nil {
+				return nil, err
+			}
+		}
 	}
 
 	return s, nil

